demoScripts/vmssQuickstart.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Quickstart - Create a Virtual Machine Scale Set with Azure CLI
+description: Get started with your deployments by learning how to quickly create a Virtual Machine Scale Set with Azure CLI.
+author: ju-shim
+ms.author: jushiman
+ms.topic: quickstart
+ms.service: virtual-machine-scale-sets
+ms.date: 11/22/2022
+ms.reviewer: mimckitt
+ms.custom: mimckitt, devx-track-azurecli, mode-api
+---
+
+# Quickstart: Create a Virtual Machine Scale Set with the Azure CLI
+
+**Applies to:** :heavy_check_mark: Linux VMs :heavy_check_mark: Windows VMs :heavy_check_mark: Uniform scale sets
+
+> [!NOTE]
+> The following article is for Uniform Virtual Machine Scale Sets. We recommend using Flexible Virtual Machine Scale Sets for new workloads. Learn more about this new orchestration mode in our [Flexible Virtual Machine Scale Sets overview](flexible-virtual-machine-scale-sets.md).
```

Comment:
Shouldn't we do the Flexible VMSS one since it is the recommended path?

Reply:
Closing this as the document is done now. But we should always do the latest recommended path, not necessarily the most common path.